src/common/services/recordToSchemaMapping/recordToSchemaMapping.interface.ts

```diff
@@ -0,0 +1,11 @@
+export type IInit = {
```

Comment:
I think the name should be more specific. It isn't obvious that it's related to a specific functionality (which I assume it is)